src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -6318,6 +6354,7 @@ namespace SurfaceGeometry {
         //       DATE WRITTEN   May 2000
         //       MODIFIED       na
         //       RE-ENGINEERED  na
+        //       ADDED WARNINGS and UPDATED DESCRIPTIONS in Sep 2022
```

Comment:
we no longer add comments like this, you can also delete the ones with na.

Reply:
Will do a final clean up after testing all the changes.